Towards A Social Platform.md

```diff
@@ -0,0 +1,88 @@
+6. Driving publication, sharing and use by setting up metrics and systems that promote friendly competition and collaboration.
+7. Providing easy to understand metrics of individual resource utilization vs. value of work to drive granting of resources. Some of these should be public. 
+8. Providing easy to understand metrics of system resource utilization, sharing value, and new “conclusions” that make it easy to ingest metrics of system success. 
+9. Providing classification, metrics and display of new public results on system. 
+
+Some of these functions are simply useful to collaborative users and laboratory leads; some of these functions are to provide justification for the system; some are to drive the principles of collaborative, knowledge driven systems biology; some are to provide incentives for using the system; some are to provide controls for misuse of the system. 
+
+## Basic Capabilities (unordered)
+
+### Verified Identity
+
+One of the the things I’d like to pursue is getting verified identity for our users. This serves the principle that users are signing up to share and publish results therefore should be identifiable. This also sets us up to minimize misuse of the system and opens some ability to open advanced scripting and arbitrary data upload since our users are, to some degree, “trusted”. 
+
+ Each account for be for a known individual except for temporary class/guest accounts (See below). ORCID, RESEARCHERID, SCOPUSID, GOOGLE SCHOLAR ID- are all possible starting points we need to determine the limits of use for all. These also allow tying in to other infrastructure that provide value to the user and perhaps ultimately to us (as we can pull it in to inform our social network.)
+
+In addition, we may need to require certain other pieces of information that allow our social functions to work. These are described in the Groups, Projects and Roles section and the Changing institutions, groups, projects. 
+### Guest and Class Accounts
+
+Because there is a privacy issue with requiring students to identify themselves and because classes and workshops may want to issue temporary accounts without long-term storage. These accounts would not be subject to identity verification. But we would need controls on when they are created and verify the overall requester (another social role: class lead). We would probably want automatic decommission dates. Maybe something to tag unreasonable usage. 
+
+### Groups, Projects and Roles
+
+One of the powerful uses of our system would be for scientific leaders to organize the work they are are overseeing and even for individual users to organize their own research. The organization is both from a management perspective and might also be from a science perspective. For example, a project may contain all Narratives and data that go into a particular paper. 
+
+#### Changing institutions/groups/projects
```

Comment:
Most of these social things have to be backed by data in one way or another. The point is that the social system is where a person builds their identity (which is represented as data about them), their relationship to others (again this is about both the groups they belong to and the common utilization of data and tools), and their relationship status (active in a given project, alumni, etc.) When they change relationships this is reflected in both their social network and in their permissions on data and narratives, etc. Therefore these things are deeply intertwined. The social part is making all of these complex relationships between one's social network and one's access and alerts easy to manage and see.